slo-monitor/src/monitors/pod_monitor.go

```diff
@@ -175,9 +183,16 @@ func (pm *PodStartupLatencyDataMonitor) Run(stopCh chan struct{}) error {
 	eventController := NewWatcherWithHandler(
 		&cache.ListWatch{
 			ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
-				options.FieldSelector = eventSelector
-				// This call is paginated by cache.Reflector.
-				return pm.kubeClient.CoreV1().Events(v1.NamespaceAll).List(options)
+				o := metav1.ListOptions{
+					Limit: 1,
+					FieldSelector: eventSelector,
```

Comment:
Actually - you don't want this FieldSelector.

Reply:
Done.

Reply:
Done.